apps/cookbook/src/app/examples/item-group-example/item-group-example.component.html

```diff
@@ -0,0 +1,40 @@
+<kirby-item-group>
+  <kirby-item>
+    <kirby-label>
+      <h3 class="kirby-text-bold">Title</h3>
+      <p detail>Detail</p>
+    </kirby-label>
+    <kirby-label slot="end">
+      <data>Value</data>
+    </kirby-label>
+  </kirby-item>
+  <kirby-item>
+    <kirby-label>
+      <h3 class="kirby-text-bold">Title</h3>
+      <p detail>Detail</p>
```

Comment:
A question! What is the `detail` part that is being applied? Is it a shorthand for class, a role, what is going on? I haven't seen it before 😄

Reply:
Using an attribute like this, is how you let the content projection in Angular know which specific slot the element should show up in (so the slot itself within the component would look something like `<ng-content select="detail"></ng-content>`).

Reply:
But, it does not actually look like the kirby-label has any detail slot, so in this case it seems like it is purely used for styling.

Reply:
We could consider using a class instead of a applying a slot to it if it is only for styling. But if we're not 100% sure if it is for only for styling, let's just leave it be for now.